### DISCUSSION TITLE
Camera switch not working for Mobile Device

### DISCUSSION THREAD

**Discussion by TeaTheDeveloper** [TeaTheDeveloper/motion-detector#30] [General]

**Comment by pablo-codes** [comment]
is now working for android

**Comment by TeaTheDeveloper** [comment]
I couldn't find a way to get it working

**Comment by TeaTheDeveloper** [comment]
But for PC, here is a fix - only one camera can be set to default.
![image](https://github.com/CovenantJunior/motion-sensor/assets/78209972/a7d4b0d0-1441-4bcd-93ee-44b0eb8bf29c)

**Reply by pablo-codes** [reply]
hmm so it keeps on taking the default camera.
i'll try and work on that

**Reply by TeaTheDeveloper** [reply]
I hope that has a fix 😃

**Reply by pablo-codes** [reply]
that i'm not sure but i will have to make researches for a long while.
